Database/Base/ShardBase.sql

```diff
@@ -313,251 +245,168 @@ CREATE TABLE `ace_position` (
   `qX` float NOT NULL,
   `qY` float NOT NULL,
   `qZ` float NOT NULL,
+  `landblock` int(5) unsigned GENERATED ALWAYS AS (conv(left(hex(`landblockRaw`),4),16,10)) STORED,
+  `cell` int(5) unsigned GENERATED ALWAYS AS (conv(lpad(substr(hex(`landblockRaw`),5,4),4,'0'),16,10)) STORED,
   PRIMARY KEY (`positionId`),
   KEY `idx_aceObjectId` (`aceObjectId`),
-  KEY `idx_landblock` (`landblockRaw`),
   KEY `idxPostionType` (`positionType`),
-  CONSTRAINT `fk_ap_ao` FOREIGN KEY (`aceObjectId`) REFERENCES `ace_object` (`aceObjectId`)
-) ENGINE=InnoDB AUTO_INCREMENT=43247 DEFAULT CHARSET=utf8;
-/*!40101 SET character_set_client = @saved_cs_client */;
-
---
--- Table structure for table `character_friends`
---
-
-DROP TABLE IF EXISTS `character_friends`;
-/*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
-CREATE TABLE `character_friends` (
-  `id` int(10) unsigned NOT NULL,
-  `friendId` int(10) unsigned NOT NULL,
-  PRIMARY KEY (`id`,`friendId`),
-  KEY `FK_character_friends_friendId_character_guid` (`friendId`)
-  /*
-  CONSTRAINT `FK_character_friends_friendId_character_guid` FOREIGN KEY (`friendId`) REFERENCES `character` (`guid`),
-  CONSTRAINT `FK_character_friends_id_character_guid` FOREIGN KEY (`id`) REFERENCES `character` (`guid`)
-  */
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-/*!40101 SET character_set_client = @saved_cs_client */;
+  KEY `idx_landblock_raw` (`landblockRaw`),
+  KEY `idx_landblock` (`landblock`),
+  KEY `idx_cell` (`cell`),
+  CONSTRAINT `fk_position_ao` FOREIGN KEY (`aceObjectId`) REFERENCES `ace_object` (`aceObjectId`) ON DELETE CASCADE
+) ENGINE=InnoDB AUTO_INCREMENT=43262 DEFAULT CHARSET=utf8;
 
---
--- Table structure for table `ace_weenie_class`
---
+/*Table structure for table `ace_weenie_class` */
 
 DROP TABLE IF EXISTS `ace_weenie_class`;
-/*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
+
 CREATE TABLE `ace_weenie_class` (
   `weenieClassId` int(10) unsigned NOT NULL,
   `weenieClassDescription` text NOT NULL,
   PRIMARY KEY (`weenieClassId`),
   UNIQUE KEY `idx_weenieName` (`weenieClassDescription`(100))
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-/*!40101 SET character_set_client = @saved_cs_client */;
 
---
--- Temporary view structure for view `vw_ace_object`
---
+/*Table structure for table `vw_ace_character` */
+
+DROP TABLE IF EXISTS `vw_ace_character`;
+
+/*!50001 DROP VIEW IF EXISTS `vw_ace_character` */;
+/*!50001 DROP TABLE IF EXISTS `vw_ace_character` */;
+
+/*!50001 CREATE TABLE  `vw_ace_character`(
```

Comment:
I contacted sqlyog support this is their response:

No, there is not.

"/*!40101 .. */;" will be treated like a comment by the server if server version is <4.0.1. If server version is > 4.0.1 the server ill treat it like an executable statement. It is simply necessary to use such 'version-dependent comments' unless the client is designed for a specific server version only (what SQLyog is not) and script generated is knon to be run only against a specific server version. You can check with 'mysqldump' or any official client by MySQL/Oracle/MariaDB and they do exactly the same.

You may refer MySQL documentation at https://dev.mysql.com/doc/refman/5.7/en/comments.html "If you add a version number after the ! character, the syntax within the comment is executed only if the MySQL version is greater than or equal to the specified version number"